packages/addons-manager/bundle/config/upstream/webhook-manifests.lib.yaml

```diff
@@ -87,6 +87,7 @@ webhooks:
         path: /mutate-run-tanzu-vmware-com-v1alpha3-clusterbootstrap
     failurePolicy: Fail
     name: clusterbootstrap.mutating.vmware.com
+    reinvocationPolicy: IfNeeded
     rules:
```

Comment:
@shyaamsn / @yharish991 - Are there other places to update for consumption by TKGS?

Reply:
If its not parametrized using values.yaml then no need to update it in other packages.